src/core/types/decimal192.rs

```diff
@@ -270,16 +613,24 @@ impl Decimal {
     /// - Panic if the number of decimal places is not within [0..SCALE(=18)]
     pub fn round_with_mode(
         &self,
-        decimal_places: i32,
+        decimal_places: u8,
         rounding_mode: RoundingMode,
     ) -> Result<Self> {
         self.native()
-            .checked_round(decimal_places, rounding_mode.into())
+            .checked_round(decimal_places as i32, rounding_mode.into())
             .ok_or(CommonError::DecimalError)
             .map(Self::from)
     }
 }
 
+#[cfg(test)]
+impl From<&str> for Decimal192 {
+    /// TEST ONLY
+    fn from(value: &str) -> Self {
+        value.parse().unwrap_or_else(|_| panic!("Test failed since the passed in str is not a valid Decimal192: '{}'", value))
```

Comment:
N.B. TEST ONLY! Very nice to have this, since we can just use "123.456" for any method/function which accepts `impl Into<Decimal>`, which allows for us to skip writing the very much more verbose "123.456".parse::<Decimal192>.unwrap().